_harp/about/_company.md

```diff
@@ -9,3 +9,14 @@ We support ourselves entirely through voluntary contributions from our alumni, t
 We believe this to be a replicable model of training that is capable of addressing acute skills shortages, widening access to work opportunities, and offering life-changing personal empowerment free at the point of delivery and without ongoing subsidies.
 
 We are partnering with a number of organisations in the social enterprise and charity sectors to provide freelancing and employment opportunities for our graduates and to open new schools in the UK and abroad.
+
+## Our hiring rate
+
+We encourage our graduates to freelance with us after they graduate, so the usual measure of graduate success, the *"hiring rate"*,  a bit misleading, but of the 52 students who studied with us in 2014/15 and who actively sought development work, all 52 are working either freelance or in permanent roles. In effect, our success rate is 100%. 
```

Comment:
I think this is too wordy. I would just go for one sentence: Our hiring rate for our graduates who have *actively sought* full time employment after the course is 100%.